api/pkg/instrumentation/metrics/metrics.go

```diff
@@ -21,6 +21,7 @@ type Collector interface {
 	// MeasureDurationMs is a deferrable version of MeasureDurationMsSince which evaluates labels
 	// at the time of logging
 	MeasureDurationMs(key MetricName, labels map[string]func() string) func()
+	RecordGauge(key MetricName, count float64, labels map[string]string) error
```

Comment:
Nitpick: it should be something like `value`, not count - it's float. Could you make this change here and in `prometheus.go` ?

Reply:
Once this is done, please feel free to merge. Thanks!

Reply:
👍🏻 Updated tests as well for consistency.